Added entries for vcluster plugin

hickey opened the pull request:
# Checklist

- [x] Provide a short description and link to your plugin.

- [x] Make sure you CI tests for your plugin and they are green.

      If you are using github, you might want to use the
      `plugin_test` action from [asdf-actions](https://github.com/asdf-vm/actions)

- [x] `asdf-plugins` CI sanity checks are green on your PullRequest.

      You can test locally using:

      ```bash
      ./test_plugin.sh --file plugins/PLUGIN_FILE
      ```


## Other Information

If there is anything else that is relevant to your pull request include that
information here.

Thank you for contributing to asdf-plugins!

hickey commented:
BTW, `test_plugin.sh` does not test if a newline terminates the plugin file. VS Code did it to me again and created the file with the repository line but no newline. Would be nice to have that check included into `test_plugin.sh`.

jthegedus commented:
>if a newline terminates the plugin file

can you elaborate on this in a new issue?

hickey commented:
Certainly. 

Testing the plugin from the command line with `test_plugin.sh` came back with no issues at all. Actually I take that back. I just changed my plugin file to not have a newline at the end of the repository line and got back:

```shell
❯ ./test_plugin.sh --file plugins/vcluster
Checking plugins/vcluster
Expected plugins/vcluster to end with a new line.
```

So the error is there just not recognizable as an error if moving quickly. I know when I have used `test_plugin.sh` in the past on other plugins that I have added that it comes back with 2 lines. In my haste and seeing 2 lines I expected everything to be fine and then when the CI on GitHub came back with an error I suspected that `test_plugin.sh` was not checking for the newline at the end of the line. My mistake. 

I would suggest that error output be prepended with "ERROR:" or "FAIL:" so that it is quite a bit more apparent when there are issues rather than have the output look like normal output. Actually I will go do a quick PR to get it moving.
